pom.xml

```diff
@@ -195,7 +195,7 @@
     <!-- org.apache.commons/commons-pool2/-->
     <commons-pool2.version>2.12.0</commons-pool2.version>
     <datanucleus-core.version>4.1.17</datanucleus-core.version>
-    <guava.version>33.2.1-jre</guava.version>
```

Comment:
After https://github.com/apache/spark/pull/42493, I think we should be able to `unify it` now ? @LuciferYang @pan3793 @dongjoon-hyun

Reply:
Ya, IIRC, that was the proposal.

Reply:
Personally, I believe that we can continue to retain this configuration way, as it may be beneficial to allow the connect module to use an independent version of Guava.

This is just my personal opinion. We can also hear what others think or discuss it in the mail list